graphlearn/CMakeLists.txt

```diff
@@ -247,6 +255,17 @@ else ()
   set (GL_KNN_FILES)
 endif ()
 
+## contrib vineyard
+if (WITH_VINEYARD)
+  message(INFO "Graph-learn is built with vineyard support.")
+  find_package (vineyard)
```

Comment:
Should we also provide vineyard installation here or in `install_dependencies.sh`?
And we should add the installation process with vineyard  support to the documentation.

Reply:
Vineyard will be available with `brew install` and `apt-get install` later.

I would update the installation docs then.